validation/joi.js

```diff
@@ -1,6 +1,6 @@
 const Joi = require("joi");
 
-const schema = Joi.object({
+const joiSchema = Joi.object({
```

Comment:
```suggestion
const userSchema = Joi.object({
```
Staraj się by nazwy były bardziej opisowe.